cli/src/main.rs

```diff
@@ -319,6 +319,30 @@ fn run() -> Result<(), CliError> {
                                 .default_value("human")
                                 .takes_value(true),
                         ),
+                )
+                .subcommand(
+                    SubCommand::with_name("default")
+                        .about("Manage default values for circuit creation")
+                        .subcommand(
+                            SubCommand::with_name("set")
+                                .about("Set a default value")
+                                .setting(AppSettings::SubcommandRequiredElseHelp)
+                                .subcommand(
+                                    SubCommand::with_name("service-type")
```

Comment:
Is there a reason why this is a separate sub command vs `set <value key> <value>` where `<value key>` is in the set `{service-type, management-type}`.  That seems more extensible, where we can add more default keys, as we decide which can be configured with defaults.

Reply:
Okay, I updated the PR to be as you suggested.